data/hulls/ship_data.csv

```diff
@@ -7,50 +7,50 @@ Arbitrator,diableavionics_arbitrator,,Diable Avionics,diableavionics_booster,,30
+Zephyr,diableavionics_zephyr,,Diable Avionics,diableavionics_splitter,,1400,100,2000,,300,0,,180,240,150,75,150,65,OMNI,,240,0.5,1,,,5,5,,,,,,,,,,,,,,,,,,,,9
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Valiant,diableavionics_valiantTrainer,Trainer,Diable Avionics,diableavionics_transitTrainer,,800,75,1200,,200,0,,160,200,200,90,180,45,OMNI,,120,0.5,1,,,3,3,,,,,,,,,,,,,HIDE_IN_CODEX,restricted,,,,,,9
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Draft,diableavionics_draft,Frigate,Diable Avionics,diableavionics_draftrepairs,4,2250,300,1500,,190,50,,120,90,60,60,60,275,NONE,,0,0,0,,,25,35,50,25,1,,10,8000,7,15,300,0.2,4,4,SHIP_WITH_MODULES,diable_ship_bp,,0.25,2,2,,13
-Arcus,diableavionics_draftarmormodule,,,,,4000,400,0,,0,0,,400,400,400,200,400,125,NONE,,,0,0,,,,,,,,,,,,,,,0,0,HIDE_IN_CODEX,module_hull_bar_only,,,,,,4
-Vapor,diableavionics_vapor,Frigate,Diable Avionics,diableavionics_evasion,6,1500,200,2400,,225,50,,175,130,110,100,80,150,OMNI,,90,0.5,1,,,20,30,35,30,1,,10,15000,10,20,210,0.2,6,6,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
-Sleet,diableavionics_sleet,Frigate,Diable Avionics,diableavionics_capacitors,5,2000,250,3000,,150,50,,140,120,100,80,80,200,OMNI,,120,0.4,1,,,30,40,80,35,1,,10,13000,8,16,240,0.2,5,5,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
+Draft,diableavionics_draft,Frigate,Diable Avionics,diableavionics_draftrepairs,4,3000,300,1500,,190,50,,120,90,60,60,60,275,NONE,,0,0,0,,,25,35,50,25,1,,10,8000,7,15,300,0.2,4,4,SHIP_WITH_MODULES,diable_ship_bp,,0.25,2,2,,13
+Arcus,diableavionics_draftarmormodule,,,,,4000,600,0,,0,0,,400,400,400,200,400,125,NONE,,,0,0,,,,,,,,,,,,,,,0,0,HIDE_IN_CODEX,module_hull_bar_only,,,,,,4
+Vapor,diableavionics_vapor,Frigate,Diable Avionics,diableavionics_evasion,6,1500,200,3000,,225,50,,175,130,110,100,80,150,OMNI,,120,0.5,1,,,20,30,35,30,1,,10,15000,10,20,210,0.2,6,6,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
+Sleet,diableavionics_sleet,Frigate,Diable Avionics,diableavionics_capacitors,5,2000,250,3000,,150,50,,140,120,100,80,80,200,OMNI,,180,0.4,0.8,,,30,40,80,35,1,,10,13000,8,16,240,0.2,5,5,,"diable_ship_bp,rare_bp",,0.5,2,2,,14
 Shear,diableavionics_shear,Tug,Diable Avionics,flarelauncher,2,450,125,750,,75,15,,135,100,100,60,30,160,FRONT,,50,0.4,1.1,,,5,5,0,10,5,,8,17000,10,30,180,0.2,5,5,CIVILIAN,diable_ship_bp,,0.3,2,2,,15
 Laminar,diableavionics_laminar,Mining Ship,Diable Avionics,maneuveringjets,2,1500,225,2000,,150,15,,115,80,60,65,40,160,OMNI,,90,0.4,1.1,,,5,10,100,40,1,,10,15000,10,30,180,0.2,2,2,CIVILIAN,diable_ship_bp,,0.5,2,2,,16
-Versant,diableavionics_versant,Hybrid Wanzer,Diable Avionics,diableavionics_transit2,8,1750,275,3500,,350,45,,110,200,200,180,220,300,FRONT,,270,0.25,0.8,,,5,5,0,0,1,,10,36000,8,25,240,0.2,8,8,,rare_bp,0.5,0.75,2,2,,17
+Versant,diableavionics_versant,Hybrid Wanzer,Diable Avionics,diableavionics_transit2,8,2000,275,4000,,350,45,,110,200,200,180,220,300,FRONT,,270,0.25,0.8,,,5,5,0,0,1,,10,36000,8,25,240,0.2,10,10,,rare_bp,0.5,0.75,2,2,,17
```

Comment:
Versant hull increased to 2000 (from 1750), Flux cap increased to 4000 (from 3500)